crates/sui-node/src/lib.rs

```diff
@@ -954,12 +954,12 @@ pub async fn build_server(
     Ok(Some(rpc_server_handle))
 }
 
-#[cfg(test)]
+#[cfg(not(test))]
```

Comment:
As an observation it makes we wonder whether we should avoid this approach overall . There might be good reason why the tests don't configure the property via the `ProtocolConfig` parameter, but seems a bit dangerous to have configs like this.